test/no-generic-verbs.test.js

```diff
@@ -0,0 +1,67 @@
+const noGenericVerbs = require('../src/rules/no-generic-verbs');
+
+let RuleTester = require('eslint').RuleTester;
+
+let ruleTester = new RuleTester();
+
+
+Object.assign(ruleTester.testerConfig, {
+	parserOptions: {
+		ecmaVersion: 2018
+	}
+});
+
+ruleTester.run('no-generic-verb', noGenericVerbs, {
+	valid: [
+		{
+			code: ``
+		},
+		{
+			code: `function watchMovie() {
+				return 'ok'
+			}`
+		},
+		{
+			code: `let number = 5`
+		},
+		{
+			code: `let obj = {
+				hi: 'hello',
+				'how': 'are',
+				'you': 'hmm'
+			}`
+		}
```

Comment:
Add tests for reading variables with generic verbs. As we disallow only definitions they should not fail

Reply:
remove variable declarations in tests for variable reading
Examples of reading tests
```
array.checkIsEmpty()
```
```
users.getFriends()
```
```
process.exit()
```
```
validateValue('dfdf')
```

Reply:
These tests doesn't contain generic verbs. But they should